apps/manifest.go

```diff
@@ -146,29 +139,16 @@ type Manifest struct {
 // Thus, custom functions to encode/decode JSON, with backwards compatibility
 // support for App and Manifest.
 func DecodeCompatibleManifest(data []byte) (m *Manifest, err error) {
```

Comment:
I would like to rename this function (and the app one) to  `UnmarshalManifest` (and `UnmarshalApp`) as they are not longer about compatibility. Thoughts?

Reply:
@levb What do you think?

Reply:
I thought about it too, as I was reviewing the code. 0/5 `Parse[Manifest|App]JSON`? But I don't think it's a must do.